src/styles/utilities/positioning.ts

```diff
@@ -11,6 +11,14 @@ export default {
     pAbsolute: {
         position: 'absolute',
     },
+    /**
+     * Web-only style.
+     */
+    pFixed: {
+        // NOTE: asserting "position" to a valid type, because it isn't possible to augment "position".
+        position: 'fixed' as ViewStyle['position'],
```

Comment:
Do we have to make assertions like this everywhere we want to use properties that we cannot argument?

Reply:
Yes, for all properties that we cannot augment we have to use assertion.